code/modules/research/xenoarchaeology/finds/finds_special.dm

```diff
@@ -150,6 +155,37 @@
 		B.blood_DNA[M.dna.unique_enzymes] = M.dna.b_type
 		M.vessel.remove_reagent(/singleton/reagent/blood,rand(25,50))
 
+/obj/item/vampiric/proc/bloodcall_final(var/mob/living/carbon/human/M)
+	if(istype(M))
+		if(istype(M.species, /datum/species/machine))
+
+			M.apply_damage(700, DAMAGE_BRUTE, BP_GROIN)
+			M.apply_damage(700, DAMAGE_BRUTE, BP_GROIN) //The way IPC limbs seem to act I needed the double damage to ensure it was removed.
+			to_chat(M, SPAN_HIGHDANGER("What have you done?! You feel like you're being ripped in half!"))
+			return
+
+		to_chat(M, SPAN_HIGHDANGER("What have you done?! You feel an incredible pull as the statue breaks. The force crushes your hands and pulls your blood into an unseen void."))
+		M.apply_damage(30, DAMAGE_BRUTE, BP_L_HAND)
+		M.apply_damage(30, DAMAGE_BRUTE, BP_R_HAND)
+		M.vessel.remove_reagent(/singleton/reagent/blood,rand(120,175))
+
+/obj/item/vampiric/proc/shatter(var/obj/item/W, var/mob/user)
```

Comment:
No single-letter parameters, give the parameter a meaningful name

Reply:
Removed the shatter proc entirely.